packages/generator-react-native-vector-icons/src/app/templates/android/src/main/java/Package.kt

```diff
@@ -0,0 +1,20 @@
+package com.reactnativevectoricons.<%= androidName %>
+
+import com.facebook.react.TurboReactPackage
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.module.model.ReactModuleInfo
+import com.facebook.react.module.model.ReactModuleInfoProvider
+
+class VectorIcons<%= className %>Package : TurboReactPackage() {
+    override fun getModule(
+        name: String,
+        reactContext: ReactApplicationContext,
+    ): NativeModule? = null
+
+    override fun getReactModuleInfoProvider(): ReactModuleInfoProvider =
+        ReactModuleInfoProvider {
+            val moduleInfos: MutableMap<String, ReactModuleInfo> = HashMap()
```

Comment:
you can replace these 2 lines with just `mapOf()`, and remove the `ReactModuleInfo` import